src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,69 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export interface Tab {
```

Comment:
Not sure why you export this interface, as far as I can see, you don't use it anywhere outside this file